src/utils/query.js

```diff
@@ -218,6 +218,21 @@ function pickQueryParams(obj) {
   ].reduce((acc, key) => (obj[key] ? { ...acc, [key]: obj[key] } : acc), {});
 }
 
+/**
+ * Parse id from string or reference
+ * @param {string|firebase.firestore.DocumentReference} value - id or reference
+ * @returns {string} Parsed id
+ */
+function parseId(value) {
+  let id;
+  if (typeof value === 'string' || value instanceof String) {
+    id = value
```

Comment:
Any reason why we can't just return here?

Reply:
it could just return